src/fn/csv-to-docs.js

```diff
@@ -1,5 +1,5 @@
 const stringify = require('canonical-json/index2');
-const uuid5 = require('uuid/v5');
+const { v5: uuid5 } = require('uuid');
```

Comment:
I particularly dislike this way of import because the left side (`v5`) looks like the name of the newly created variable while in reality the new variable that holds the imported value is the right name (`uuid5`) in the expression. Import statements with the `as` operator are more clear on this, but we cannot use that syntax outside typescript files unless we drop older versions of Node support I guess.